source/code/source/playfab/PlayFabIXHR2HttpPlugin.cpp

```diff
@@ -1,5 +1,6 @@
 #include <stdafx.h>
```

Comment:
you may move stdafx include under the protection block

Reply:
I disagree Vicodex. In this case, you can't. It'll error if #include <stdafx.h> is not present, and the first code line, even if there's nothing else in the .cpp file.